src/components/interactive-builder/add-question.modal.tsx

```diff
@@ -195,6 +201,21 @@ const AddQuestionModal: React.FC<AddQuestionModalProps> = ({
       }),
     );
   };
+
+  const handleOrdersSelect = (concept: Concept) => {
+    const updatedDatePickerType = getDatePickerType(concept);
+    if (updatedDatePickerType) setDatePickerType(updatedDatePickerType);
```

Comment:
The datepicker type is unnecessary
```suggestion
```

Reply:
Also, prefer wrapping the `if` block in braces.